### PR TITLE
Removed redundant operators in Expression

### PR DESCRIPTION
**Pull request by ayusham001**
Dropped the following operators from the expression —

 - [x] Context
 - [x] Literal
 - [x] EqualTo
 - [x] Input

/claim #330

### CODE REVIEW

File: src/lambda/expression.rs [hunk 1]
```diff
@@ -13,118 +13,96 @@ use crate::json::JsonLike;
...
+#[derive(Clone, Debug)]
+pub enum Expression {
+    Unsafe(Operation),
+}
+
+#[derive(Clone, Debug)]
+pub enum Context {
+    Value,
+    Path(Vec<String>),
 }
```
Comment (tusharmath) [inline src/lambda/expression.rs:108]: Too much whitespace changes. Is it manually performed or some tool was used?

Comment (ayusham001) [reply src/lambda/expression.rs:108]: I have formatted the code.

Comment (tusharmath) [reply src/lambda/expression.rs:108]: How did the order of things change automatically?

Comment (ayusham001) [reply src/lambda/expression.rs:108]: I have utilized a VS Code extension for formatting, and I can undo it if necessary.

File: src/lambda/lambda.rs [hunk 2]
```diff
@@ -1,128 +1,137 @@
...
-  }
+    pub fn context_path(path: Vec<String>) -> Self {
+        Lambda::new(expression::Expression::Context(expression::Context::Path(path)))
+    }
 
-  pub fn from_request_template(req_template: RequestTemplate) -> Lambda<serde_json::Value> {
-    Lambda::new(Expression::Unsafe(Operation::Endpoint(req_template)))
-  }
+    pub fn from_request_template(req_template: RequestTemplate) -> Lambda<serde_json::Value> {
+        Lambda::new(expression::Expression::Unsafe(Operation::Endpoint(req_template)))
```
Comment (tusharmath) [inline src/lambda/lambda.rs:58]: We can drop the lambda functions that were referencing the deprecated expressions.

**Comment by neo773** [comment]
Seeing a bunch of errors


image

**Comment by tusharmath** [comment]
Same, it just shouldn't work but weird that my build is passing!

File: src/lambda/expression.rs (hunk 1 above)
Comment (tusharmath) [reply src/lambda/expression.rs:108]: can you run `./lint.sh` present in the root folder?

Comment (ayusham001) [reply src/lambda/expression.rs:108]: did

**Comment by tusharmath** [comment]
@ayusham001 Are you still on this or can I release this issue for someone else to pick?

**Comment by ayusham001** [comment]
> @ayusham001 Are you still on this or can I release this issue for someone else to pick?

Hi @tusharmath rn m cancelling my attempt.
Thank you.

**Pull request closed without merging**